### ISSUE TITLE
2018-04-27 Public User Feedback Meeting - Enterprise Use Case

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#52]
# 2018-04-27 Public User Feedback Meeting - Enterprise Use Case

## Time

Friday, 2018-04-27 at 17:00 UTC / noon ET / 11am CT / 9am PT

Followed immediately afterward by 20 minute private discussion at 18:00 UTC / 1pm ET / noon CT / 10am PT.

## Links

* **Minutes Doc**: TBD
* **Previous Minutes Doc**: TBD

## Agenda

This is a public User Feedback meeting with [invited participants](https://github.com/nodejs/user-feedback/issues/38) focused on specific topics. Please feel free to propose discussion items for the agenda related to this topic below. Create a new issue to propose a new topic. Normally, @nodejs/user-feedback [team meetings](https://github.com/nodejs/user-feedback/labels/neuf-meeting) are extracted from [neuf-agenda](https://github.com/nodejs/user-feedback/labels/neuf-agenda) labelled issues and pull requests from the [user-feedback](https://github.com/nodejs/user-feedback) repo prior to the meeting.

* Topics:
  - Round table Introductions:
    - Who are you?
    - Where do you work?
    - What do you do?
    - Describe your Node.js. Enterprise Use Case
  - What's working and what isn't for Node.js in the Enterprise?
    - ✋ _Raise hand to join the discussion_
* Open feedback forum

## Invited
* User Feedback team members
  * Dan Shaw (@dshaw - User Feedback Champion, Mentorship, CommComm)
  * Mihai Ene-Pietrosanu (@mihaiep, User Feedback)
  * Michael Dawson (@mhdawson - User Feedback, CommComm, TSC Chair)
  * Tierney Cyren (@bnb - User Feedback, CommComm Chair)

* Enterprise Use Case Representatives
  * Ahmad Nassri (@ahmadnassri - Principal Architect, TELUS)

## Public participation

The Node.js project live streams meetings so anyone can tune in. Everyone is invited to participate in the live chat. The stream will start playng at https://www.youtube.com/c/nodejs+foundation/live once we go live.

There's usually a short cat-herding time at the start of the meeting and occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and we'll be online shortly. ❤️

## Joining the meeting

Join link for invited members and participants: https://zoom.us/j/353642832

There is a standing invitation for all @nodejs/TSC and @nodejs/community-committee members to attend in observer capacity.

**Comment by mihaiep** [comment]
Meeting notes [here.](https://docs.google.com/document/d/1909UvdMuMJqFb0ONmbCGGjEcyfdpx8Vx7g-L9Aedq4Y/edit#) . md file will be ready in PR.

**Comment by dshaw** [comment]
@bnb Sounds like a good idea to me.

**Comment by dshaw** [comment]
Action items:
* What are can Enterprise deployments do to engage with a Current Release and provide better feedback?
  - Where does that feedback go?
  - What are the tests or scripts folks can use to provide feedback?

**Comment by jonathan-casarrubias** [comment]
Wow such a bad luck of mine, just accidentally found this... I'm actually working on building an Enterprise Grade Framework for Node.JS called [OnixJS](https://github.com/onixjs)... Me and my team are trying to fix several issues related to the Node.JS community related to enterprise problems.

From over-dependency in open-source modules, like when everybody was unable to deploy because of the lame pad-left issue we all remember, but also providing features and services that most of the current NodeJS frameworks won't provide, though you can see those features in other languages like java and .net.

I hope to be part of the following up.

Regards
Jon

**Comment by mhdawson** [comment]
@jonathan-casarrubias hope to see you in the next meeting

**Comment by jonathan-casarrubias** [comment]
@mhdawson Cool, I'll be looking forward to that meeting next Thursday.

Cheers
Jon

**Comment by mhdawson** [comment]
@jonathan-casarrubias, we have 3 groups on the go, general, tooling and enterprise so next week will be general, however, you are still welcome.

**Comment by jonathan-casarrubias** [comment]
@mhdawson sure no problem I'm still interested in general and tooling since a big portion of my efforts has been contributions to the open-source community in terms of tooling, etc...

Anyway, we agree that important meeting for me will be the enterprise one.

Thanks for clarifying

Cheers
Jon

**Comment by SarahKConway** [comment]
when is the next meeting? Are there notes from previous meetings?

**Comment by mhdawson** [comment]
@mihaiep I think you are working on the minutes?

As for the next meeting I think we might have been thinking 2 weeks after next Friday, but don't think its settled yet.  @dshaw

**Comment by mihaiep** [comment]
All User Feedback minutes/notes are [here](https://github.com/nodejs/user-feedback/tree/master/meetings).  If you are talking about [notes](https://docs.google.com/document/d/1kswsnzh5ziblp3mcrKS8S3_UPGo7EK5MLybIoDCqxDM/) of May 11 will be available shortly via PR, and the recording of May 11, 2018 is [here](https://www.youtube.com/watch?v=eiN0gnAJNeU&t=10s).
Next User feedback meeting is on May 25. Hoe that helps, thanks.
